_articles/bluetooth.md

```diff
@@ -13,8 +13,68 @@ section: network-troubleshooting
+
+
+## Device Specific Differences
+
+Every Bluetooth device is different. They use the same or similar protocols, but the printed circuit boards (PCBs), are specific to each device, and the firmware they are running is often custom designed, and closed-source.
+
+Some of the code and technologies that make Bluetooth work reliably are patented, and only device vendors who have licensed the permission to use that patented technology will be able to experience the full benefits (see "audio input/output" section below.)
+
+
+## Kernel Versions
+
+As the Linux kernel develops, support for more devices are added. Sometimes Bluetooth devices will work better in a different kernel version.
+
+
+## OS Versions
+
+Similar to the kernel versions. Improvements are often made in newer versions of Ubuntu and Pop!\_OS. Running software updates is always a good idea, followed by a reboot.
+
+<!--## Bluez Versions-->
+
+## Configuration Issues
+
+Sometimes Bluetooth devices are working correctly, but something in settings needs to be reset.
+
+The easiest way to test this is to "forget" the paired Bluetooth device, and pair it again.
+
+A more thorough way of testing this would be to create a [test user](https://support.system76.com/articles/other-accounts/), or boot from a [Live Disk](https://support.system76.com/articles/live-disk/) to see if Bluetooth works in either case. 
+If it does, config files may need deleted. If it doesn't (especially in the Live Disk), reinstalling the OS may solve the problem. 
+Reinstalling the OS won't affect Bluetooth hardware directly, but resetting and starting with a clean slate can solve a slew of problems and save time hunting for a specific file or bug.
+
+
+# Setting Expectations
+
+Because of all of these factors, if the steps outlined in the Bluetooth troubleshooting article, and the previous troubleshooting steps don't resolve the issue, the issue may not be resolved at all. 
+Or, in a future update or change to the system, the devices may start working again. In some cases (many cases) users will not experience any issue with Bluetooth at all.
+
+## Audio Input/Output
+
+Bluetooth audio devices, such as headphones, usually default to the A2DP protocol, which works effectively as an audio output source.
+
+Bluetooth devices with microphones built in, can be used if the device supports HFP/HSP. However, without the technology that companies like Sony have patented, the solution is to divide up the audio stream so that some of it is used for audio out and some for audio in. 
+This process lowers the sound quality of the stream when in HSP/HFP mode, so audio may be "tinny" or at a lower volume. That is expected behavior.
```

Comment:
tinny or compressed
(it has less fadelity)

Reply:
Added.